crates/sargon/src/profile/logic/profile_next_derivation.rs

```diff
@@ -67,7 +67,7 @@ impl Profile {
         kind: EntityKind,
         network_id: NetworkID,
         factor_source_id: FactorSourceIDFromHash,
-    ) -> HDPathValue {
```

Comment:
these methods are soon to be deleted, replaced by the FactorInstancesProvider in a coming PR.